### PR TITLE
Kh 141622021 calorie sums

### PR DESCRIPTION
**Pull request by kheppenstall**
@Sh1pley PR covers the 6 active stories relating to summing and finding the remainders of the calories (and changing their color based on their value). Ready for review! Right now total calories is hard coded since I figured we'll save all of this data in the browser somehow tomorrow and we can sum it up then.

@case-eee Would love some feedback on how we're organizing our javascript (check out index.js).

screen shot 2017-03-13 at 5 58 58 pm

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,6 +1,60 @@
...
 $("document").ready(function() {
   var time = moment();
   $('.current-day section').html(time.format("dddd, MMMM Do YYYY"));
-})
\ No newline at end of file
+
+  calculateCalorieSums();
+  populateTotalsTable();
+  addColorToRemainingCalories();
+})
```
Comment (Sh1pley) [inline lib/index.js:8]: ^ this is awesome

File: lib/index.js [hunk 2]
```diff
@@ -1,6 +1,60 @@
...
+
+function calculateCalorieSums() {
+  calorieSum(400, 'breakfast');
+  calorieSum(200, 'snack');
+  calorieSum(600, 'lunch');
+  calorieSum(800, 'dinner');
+}
+
+function calorieSum(goal, meal) {
+  var tag = '#' + meal + '-list';
```
Comment (Sh1pley) [inline lib/index.js:18]: wizardry!

File: lib/index.js [hunk 3]
```diff
@@ -1,6 +1,60 @@
...
+    if(!isNaN(value) && value.length != 0) {
+      sum += parseFloat(value);
+    }
+  });
+  return sum
+}
+
+function addColorToRemainingCalories() {
+  $('.remaining-count').each(function() {
+    addGreenOrRed($(this));
```
Comment (Sh1pley) [inline lib/index.js:37]: I like it. functions in functions

**Review by Sh1pley — APPROVED** [review]
really like it man, I need to rearrange my mind and get on this function building party.

File: lib/index.js (hunk 2 above)
Comment (case-eee) [reply lib/index.js:18]: 🎉

**Comment by case-eee** [comment]
@kheppenstall @Sh1pley 👍

**Pull request merged**